src/pybind/mgr/dashboard/controllers/settings.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from contextlib import contextmanager
+
+import cherrypy
+
+
+from . import ApiController, RESTController
+from ..settings import Settings, Options
+from ..security import Scope
+
+
+@ApiController('/settings', Scope.CONFIG_OPT)
+class SettingsController(RESTController):
```

Comment:
Shouldn't we rename this controller to `Settings`, for consistency in the API documentation?

![screenshot_20180809_160142](https://user-images.githubusercontent.com/14297426/43907768-2b308d16-9bee-11e8-9028-47db0815ac8d.png)